sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -325,7 +335,13 @@ export class EventHubClient {
       throw new Error("'connectionString' is a required parameter and must be of type: 'string'.");
     }
     const config = EventHubConnectionConfig.create(connectionString, path);
-
+    if (options && options.webSocket) {
+       config.webSocket = options.webSocket;
+       config.webSocketEndpointPath = "$servicebus/websocket";
```

Comment:
We should set `config.webSocketEndpointPath` even for browser scenario where user is not expected to provide the web socket

Reply:
I would say that we can remove the if checks here...

```javascript
config.webSocket = options && options.webSocket;
config.webSocketEndpointPath = "$servicebus/websocket";
config.webSocketConstructorOptions = options && options.webSocketConstructorOptions
```